### PR TITLE
Remove useless content (#76)

### LINKED ISSUE #76: Remove Unused Import statements and variables
Lot of files have some unused import statements and comments generated by android studio when the file is created, these should be removed.

### PR DESCRIPTION
**Pull request by AkshayCHD**
fixes #76

### CODE REVIEW

**Comment by 4lch4** [comment]
Out of curiosity, will these PR's work fine with the latest build tools as I updated in my PR #73? Granted, I still have to figure out what's wrong with Travis but after that's worked out, will it cause any issues with the PR's you've been submitting, @AkshayCHD?

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java [hunk 1]
```diff
@@ -40,10 +41,6 @@
 import retrofit2.Callback;
 import retrofit2.Response;
 
-/**
```
Comment (4lch4) [inline app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]: Should we be removing the javadocs that give credit to whoever created the class though, even if it's boilerplate from Android Studio?

Comment (4lch4) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]: We could update them to all follow a standard date format and have more information regarding the class so it's not _just_ the creator information.

Comment (4lch4) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]:
```java
/**
 * The {@link AppCompatActivity} class responsible for displaying {@link Droplet} information.
 *
 * @author the-dagger on 11/26/2016
 */
```

Maybe something like this?

Comment (AkshayCHD) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]: @Alcha yes as I have only removed some stuff that was not needed so i believe there won't be any issue, even if there is, the conflicts for such PRs could be resolved easily. 
About removing the creator comments, they are quite inconsistent as the name mentioned there might not coincide with our real name or our github username and also just giving credit to the one who created that file and not to the ones who maintain and enhance it is not correct in my opinion.

**Comment by AkshayCHD** [comment]
@the-dagger @championswimmer please have a look

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java (hunk 1 above)
Comment (4lch4) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]: Then we can change it to show who created it and then who now maintains it. Especially if the majority of the code remaining is still written by the original author.

Comment (4lch4) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]:
```java
/**
 * The {@link AppCompatActivity} class responsible for displaying {@link Droplet} information.
 *
 * @author created by the-dagger on 11/26/2016, maintained by AkshayCHD
 */
```

Comment (AkshayCHD) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]: @Alcha currently the project is small but still we have alot  of people who contributed in the past and there would be many who would be contributing in the future. I see no use of maintaining this record when all the information about each and eery contribution is avaliable on github.

Comment (4lch4) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]: Fair enough, it's a difference of opinion then.

In that case, we'll have to leave it to @the-dagger and @championswimmer to decide.

Comment (AkshayCHD) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java:43]: Yaa, it is better to leave it to them

**Comment by championswimmer** [comment]
Please make some constructive contributions from the future. Not just for the sake of raking some BOSS bounties or green streaks on Github. This is the most pointless PR I have seen till now. Especially with all the authorship removal.

**Comment by AkshayCHD** [comment]
@championswimmer sorry you felt that way, i had no such intentions, there were some unused import statements and variables that were there in the code so i thought that they need to be removed, and about the authorship comments, I contributed in a project at FOSSASIA, there whever i made a pr the maintainer asked me to remove authorship comments very first so i thought same would apply here.
But are't the unused statements and variables supposed to be removed.

**Pull request closed without merging**